docs/sources/reference/cli/environment-variables.md

```diff
@@ -14,13 +14,28 @@ The following environment variables are supported:
 
 * `GODEBUG`
 * `HTTP_PROXY`
+* `HTTPS_PROXY`
+* `NO_PROXY`
 * `PPROF_MUTEX_PROFILING_PERCENT`
 * `PPROF_BLOCK_PROFILING_RATE`
 * `GOMEMLIMIT`
-
+* `AUTOMEMLIMIT`
+* `GOGC`
+* `GOMAXPROCS`
+* `GOTRACEBACK`
 
 Refer to the [Go runtime][runtime] documentation for more information about Go runtime environment variables.
 
+# Windows Service Configuration
```

Comment:
```suggestion
## Windows Service Configuration
```

We can only have one H! heading per topic.

Also... do we really need this Windows-specific info here? What about Linux and MacOS? If we add specific instructions for Windows we should consider the same for the other supported OSes.

Reply:
Removed the windows specific information for now.

Reply:
I added this to a liked issue so I don't forget to eventually add the Env Var details.